nuget/Auth0.OidcClient.WPF.nuspec

```diff
@@ -12,12 +12,15 @@
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>Auth0 OIDC Client for WPF apps</description>
     <releaseNotes>
+      Version 3.1.4
+      - Now supports .NET Core 3.1 projects
+
       Version 3.1.3
       - WPF version is now actually strong-named.
 
       Version 3.1.2
       - Allow ID tokens "issued at" (iat) claims from "the future" to allow
-        slow local clocks on mobile and desktop devices.
```

Comment:
Can you revert these changes. I know they shouldn't be space sensitive which is why VS reformats them but they actually are in NuGet.

Reply:
I will close this pull request and submit a new one without the whitespace changes